src/operations.rs

```diff
@@ -125,10 +123,9 @@ pub(crate) fn aesenc(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
-    use core::mem::transmute;
```

Comment:
Maybe #170 is caused by this?

Reply:
Looks like it. A fix has been uploaded: https://github.com/tkaitchuck/aHash/pull/171

Reply:
That we accidentally left one `core::mem::transmute` call in demonstrates to me that the name of `zercopy::transmute!` makes its use error-prone unless the `transmute!` call is moved out of the `unsafe` block (like in #171), as otherwise it is too easy to call `mem::transmute` instead, accidentally.